FizzBuzz/UI - multiple parameters.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'logic'
+
+class UI_multiple_parameters
+  def initialize
+    @fb_case = Fizz_Buzz.new
+  end
+
+  def fizz_buzz_input(input, fb_case)
+    begin
+      input = Float(input)
+    rescue ArgumentError => e
+      puts "#{input} is an improper value: #{e.class.name}, try again."
+      is_float = false
+    end
+
+    if is_float == false
```

Comment:
it doesn't seem like we're doing much in those empty `if` blocks like this one or `input%1 == 0` below, so might be a good idea to get rid of them.

Reply:
I'm not sure if I know how to do it without crippling the whole thing, but I'm on it. Thanks!

Reply:
just check if `is_float` is true instead :)